cmd/config.go

```diff
@@ -54,17 +54,20 @@ func configFlagSet() *pflag.FlagSet {
 	flags.Bool("no-usage-report", false, "don't send anonymous stats to the developers")
 	flags.Bool("no-thresholds", false, "don't run thresholds")
 	flags.Bool("no-summary", false, "don't show the summary at the end of the test")
+	reportSummaryUsage := "output summary to file in json format, empty string means output to standard out"
```

Comment:
I would prefer if this is directly in the `flags.StringP` call below, maybe just move it to the next line as :
```
 	flags.StringP("report-summary", "r", "", 
          "output summary to file in json format, empty string means output to standard out") 
```